hoofdstuk_3_oefeningen.tex

```diff
@@ -58,100 +64,62 @@ \subsubsection*{b)}
+$W_4$ is een deelruimte van $\mathbb{R}^{3 \times 3}$. Ze moet aan de 3 eigenschappen voldoen.
 \begin{itemize}
-\item De nulmatrix $O \in W_5$
-\item $A,B \in W_5 \longrightarrow A+B \in W_5$
+\item De nulmatrix $O \in W_4$
+\item $A,B \in W_5 \longrightarrow A+B \in W_4$
 \[
 \sum\limits_{i=1,j=1}^a (A+B)_{ij} = \sum\limits_{i=1,j=1}^a A_{ij} + \sum\limits_{i=1,j=1}^a B_{ij} = 0+0 = 0
 \]
-\item $ A \in W_5, \lambda \in \mathbb{R} \longrightarrow \lambda A \in W_5$
+\item $ A \in W_4, \lambda \in \mathbb{R} \longrightarrow \lambda A \in W_4$
 \[
 \sum\limits_{i=1,j=1}^a (\lambda A)_{ij} = \lambda \sum\limits_{i=1,j=1}^a A_{ij} = \lambda 0 = 0
 \]
 \end{itemize}
-\subsubsection*{f)}
-$W_6$ is geen deelruimte van $\mathbb{R}^{3}$ omdat de nulmatrix $O \not \in W_6$ aangezien $\det(O) = 0$.
 
-\subsubsection*{g)}
-$W_7$ is een deelruimte van $\mathbb{R}^{3x3}$.
+\subsubsection*{e)}
+$W_5$ is geen deelruimte van $\mathbb{R}^{3x3}$ omdat de nulmatrix $O \not \in W_5$ aangezien $\det(O) = 0$.
+
+\subsubsection*{f)}
+$W_6$ is een deelruimte van $\mathbb{R}^{3x3}$.
 \begin{itemize}
-\item De nulmatrix $O \in W_7$, de nulmatrix is immers ook een symmetrische matrix.
+\item De nulmatrix $O \in W_6$, de nulmatrix is immers ook een symmetrische matrix.
 \item De som van twee symmetrische matrices is ook een symmetrische matrix.
 \item Een symmetrische matrix vermenigvuldigen met een scalar geeft opnieuw een symmetrische matrix.
 \end{itemize}
 
-\subsubsection*{i)}
-$W_9$ is geen deelruimte van $\mathbb{R}^{3\times 3}$. De optelling is namelijk niet intern.
-\[
-(0,1,0) + (1,0,0) = (1,1,0) \not \in W_9
-\]
+\subsubsection*{g)}
+$W_7$ %TODO: 3.6 oef 2
 
-\subsubsection*{j)}
-$W_{10}$ is geen deelruimte van $f:\mathbb{R}\rightarrow\mathbb{R}$.\\
-Tegenvoorbeeld: we maken een functie $g \in W_{10}$ met $g : \mathbb{R}\rightarrow\mathbb{R} : x \mapsto e^x$.
+
+\subsubsection*{h)}
+$W_{8}$ is geen deelruimte van $f:\mathbb{R}\rightarrow\mathbb{R}$.\\
+Tegenvoorbeeld: we maken een functie $g \in W_{8}$ met $g : \mathbb{R}\rightarrow\mathbb{R} : x \mapsto e^x$.
 \[
-(g+g)(0) = g(0) + g(0) = 1+1 = 2 \Longrightarrow (g+g) \not \in W_{10}
+(g+g)(0) = g(0) + g(0) = 1+1 = 2 \Longrightarrow (g+g) \not \in W_{8}
 \]
+\subsubsection*{i)}
+$W_9$
 
-\subsubsection*{l)}
-$W_{12}$ is een deelruimte van $\mathbb{R}^{3\times 3}$.\\
+\subsubsection*{j)}
+$W_{10}$ is een deelruimte van $\mathbb{R}^{3\times 3}$.\\
 Het is duidelijk dat $0 \in W_{12}$.\\
 Neem $f, g \in W_{12}$ en $\lambda, \mu \in \mathbb{R}$ willekeurig. Omdat $f$ en $g$ integreerbaar zijn, is $\lambda f + \mu g$ integreerbaar.
 \begin{align*}
   \int^1_0 (\lambda f + \mu g)(x)dx &= \int^1_0(\lambda f(x) + \mu g(x))dx\\
   &= \int^1_0 \lambda f(x)dx + \int^1_0 \mu g(x)dx\\
   &= \lambda \int^1_0 f(x)dx + \mu \int^1_0 g(x)dx = \lambda \cdot 0 + \mu \cdot 0 = 0
 \end{align*}
+\subsubsection*{jk}
+$W_{11}$
 
-
-
-\subsection{Oefening 3}% deze oplossing was fout, deze versie is juist
```

Comment:
Is er een specifieke reden dat deze oefening weg is?

Reply:
Deze oefening staat niet meer in de nieuwe cursus